### ISSUE TITLE
Rule idea: omit parentheses in argumentless mixin calls

### ISSUE THREAD

**Issue by dryoma** [kristerkari/stylelint-scss#23]
Omit parentheses in argumentless mixin calls. Inspired by [Sass Guidelines](https://sass-guidelin.es/#argument-less-mixins). Just in case any codestyle would want this.

```css
// Good:
.foo {
  @include center;
}

// Bad:
.foo {
  @include center();
}
```

**Comment by kristerkari** [comment]
Thanks! This should be easy to implement.

**Comment by dryoma** [comment]
Implemented in 9d8fdbb84ebd74c3aa73a7ff75d36e5bfb3ed9be
